app/components/agenda/agendaitem/send-to-vp-modal.js

```diff
@@ -96,25 +215,31 @@ export default class SendToVpModalComponent extends Component {
     this.args?.onClose();
   });
 
-  async loadDocumentTypes() {
-    this.documentTypes = await this.conceptStore.queryAllByConceptScheme(
-      CONSTANTS.CONCEPT_SCHEMES.DOCUMENT_TYPES
-    );
+  @action
+  onChangeComment(event) {
+    this.comment = event.target.value;
+  }
 
-    const BESLISSINGSFICHE = this.documentTypes.find(
-      (type) => type.uri === CONSTANTS.DOCUMENT_TYPES.BESLISSINGSFICHE
+  async loadDocumentTypes() {
```

Comment:
This loads the same (or at least an overlapping set of) document types as the `loadData` function. I'd rather see this in a separate task, with the constants then being accessible for the whole component all at once.